src/app/pages/impact/section/founding.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Section, HeadingAndDescription} from './section';
+import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
+import './founding.css';
+
+export default function Founding({heading, description, portrait, caption}) {
+    return (
+        <Section id="founding">
+            <div className="text-overlapping-photo">
+                <img src={portrait.image} alt={portrait.altText} />
+                <HeadingAndDescription
+                    classList={['text-block', 'card']}
```

Comment:
does the array need a `.join(' ')` call?

Reply:
No, the join happens in the component.